src/app/create-event/page.tsx

```diff
@@ -95,11 +92,11 @@ export default function CreateEvent() {
         console.error('Unexpected error:', error)
       }
     }
-
-    // Call handleSaveResponse to save the response
-    handleSaveResponse()
+    // handleSaveResponse() // Call handleSaveResponse to save the response
```

Comment:
Is this commented out for a reason? Is the UI working but the save system still broken?

Reply:
i'm assuming this will be returned after the backend integration?

Reply:
yes, left it commented out until we have the backend implemented with this